airbyte-integrations/bases/standard-destination-test/src/main/resources/exchange_rate_catalog.json

```diff
@@ -5,13 +5,13 @@
       "json_schema": {
         "properties": {
           "date": {
-            "dataType": "string"
```

Comment:
This change was sneaky!

Are we supposed to handle `datatype` fields in the JSON Schema in addition to `type` ?

Reply:
yuck! sorry. no that was an old iteration of the catalog that used `datatType`. does not not need to be supported anymore.